components/mailbox/src/Mailbox.svelte

```diff
@@ -655,7 +655,7 @@
             </div>
           {/if}
           {#if selectedThreads.size}
-            {#if actions_bar.includes("delete")}
+            {#if actions_bar.includes("delete") && false}
```

Comment:
effectively removing delete

Reply:
From Oliver:

the way we would do delete is move the email to a folder with a "trash" label
the only issue is that each email provider has a different label for what "trash" is...
so we'd have to try each provider in the middleware to delete